core/app/models/spree/tax_rate_category.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  class TaxRateCategory < Spree::Base
+    self.table_name = "spree_tax_rates_categories"
```

Comment:
If you name the table `spree_tax_rate_categories` then I think you won't need this.

Though I'd say maybe let's just name this `TaxRateTaxCategory` with a table name of `spree_tax_rate_tax_categories`.  That seems like that'd be the default rails way of naming it, even though it's a little verbose.